.github/workflows/stale-issue-needs-info.yml

```diff
@@ -1,4 +1,4 @@
-name: "Close stale issues that requires info"
+name: "Mark issues stale that requires info"
```

Comment:
I think this should say "Mark issues stale that require info".

Reply:
:+1:  Updated.